package.json

```
@@ -1,16 +1,18 @@
 {
-  "name": "drinkingcast",
-  "version": "0.0.1",
-  "description": "A drinking game using airconsole",
-  "scripts": {},
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/b9a9s9/DrinkingCast.git"
+  "name": "DrinkingCast",
+  "version": "1.0.0",
```

Ik had in gedachte om versie nummers als volgt te doen(template 0.0.0):

Eerste getal: publieke release
Tweede getal: development release (als er een nieuwe grote functionaliteit is toegevoegd bijvoorbeeld)
Derde getal: release waar een deel van een (grote) functie is toegevoegd

Met deze logica zou 0.1.0 of 0.0.1 een goed versie nummer zijn.

What do you think?

---

Sounds good to me

---

Top. Pas je dat dan nog aan? 
```suggestion
  "version": "0.0.1",
```

---

yes en dan gewoon pushen?

---

ja. alle pushes die je in je branch doet, worden geupdate in je PR. (dus je kan mijn suggestion gebruiken. maar als je meerdere changes hebt is t vaak makkelijker om in je editor te doen.)

zoals nu bijv. moet je dit, en de package-lock aanpassen, dat kan in principe makkelijk in 1 commit.